modules/ROOT/pages/depl-examples/ubuntu-compose/ubuntu-compose-hetzner.adoc

```diff
@@ -25,6 +25,8 @@ NOTE: With the minimum configuration available on Hetzner, you can start with ab
 
 NOTE: This guide references the latest {version-type} version of Infinite Scale.
 
+NOTE: With this setup, each deployment can only contain one instance of Infinite Scale.
```

Comment:
```suggestion
NOTE: With this setup, each deployment only contains one instance of Infinite Scale.
```
Nit-picking: but one can actually create a deployment with multiple instances with lots of effort.